03-task-manager/starter/controllers/tasks.js

```diff
@@ -0,0 +1,31 @@
+const Task = require('../models/Tasks')
+
+const getAllTasks = (req,res) =>{
+    
+    res.send('Todos los Items')
+}
+
+const CreateTask = async (req,res) =>{
```

Comment:
```suggestion
const createTask = async (req,res) =>{
```
Probably want to lowercase this to match the others